docs/tokenomics/token-holders/nns-app-quickstart.md

```diff
@@ -264,6 +264,50 @@ To disburse a neuron and return its ICP utility tokens:
 
 - #### Step 4:  Verify the transaction information, then click **Confirm and Send**. 
 
+## Neuron following
+
+To maximize neuron rewards and fully utilize the NNS's liquid democracy, neuron owners are encouraged to vote on every proposal submitted to the NNS. However, it may be hard for neuron owners to manually vote on every single proposal submitted, since several proposals may be submitted at one time, or neuron owners may not have the necessary level of technical understanding to evaluate proposals themselves. To aid in this, the NNS supports **neuron following**.
```

Comment:
```suggestion
To maximize neuron rewards and vote on every proposal submitted to the NNS, neuron holders can make use of the NNS's liquid democracy. It may be hard for neuron owners to manually vote on every single proposal submitted or neuron owners may not have the necessary level of technical understanding to evaluate proposals themselves. To aid in this, the NNS supports **neuron following**.
```